### ISSUE TITLE
Molten Alumite has multiple conflicting mixture amounts

### ISSUE THREAD

**Issue by stiffchub** [Krutoy242/Enigmatica2Expert-Extended#139]
PlusTiC recipes for alloying alumite results in different amounts of alumite with the same recipe.

JEI shows 5 mB of aluminum, 2 mB of obsidian, and 2 mB of iron alloys into 3 mB of alumite for one recipe.
The other recipe has 2 mB of iron, 5 mB of aluminum, and 2 mB of obsidian giving only 1 mB of alumite.

The last two recipes, that create larger amounts, are based off the 3 mB result.

**Comment by Krutoy242** [comment]
This recipes added by other mods. Sadly, i cant remove them.

Still, its not big problem, because when you try alloy Alumite, "right" recipe with better output always used first.

**Comment by stiffchub** [comment]
I just tested it in creative mode. Put in 2 obsidian blocks for 288 mB, 2 iron ore blocks for 288 mB, and 5 aluminum ore blocks for 720 mB, and it alloyed into 228 mB of alumite. It should give 432 mB.
Putting in twice the amount gave me 660 mB, 432 mB more
Putting in three times the amount game me 1092 mB, 432 more again.

I also tested it with various amounts alumite already in the smeltery and always got 228 mB of alumite from what should be the first 432 mB.

I'm manually putting them into the smeltery controller.

Seems like only the first 3 ingots worth get incorrectly alloyed.
